### DISCUSSION TITLE
Rollout in the helm install order

### DISCUSSION THREAD

**Discussion by Purushotham233** [argoproj/argo-rollouts#3364] [General]
Helm sorts all the manifests/entity while install or uninstall and apply them in that order. Incase of Service, Deployment as per this [documentation](https://github.com/helm/helm/blob/e81f6140ddb22bc99a08f7409522a8dbe5338ee3/pkg/releaseutil/kind_sorter.go#L31), first service is applied and then deployment. 

We are moving all our deployment objects to rollout objects. Do we know where does the Rollout comes in this order?

**Comment by kostis-codefresh** [comment]
Judging by this https://github.com/helm/helm/issues/7580 Rollouts are installed after everything else.
